### PR TITLE
updated readme with route info, add more tools, alphabetical order for tools

### LINKED ISSUE #61: ReadMe cleanup
Small readme cleanup under contact:
```
Heather Bisgaard - @vbchomp
Charlie Fadness - @fadnesscharlie
Tom McGuire - @MuckT
Jeremy Brazell - @jeremywbrazell
```

### PR DESCRIPTION
**Pull request by fadnesscharlie**
- [x] Added info on Each Route 
- [x] Added more tools we used
- [x] Order the tools we used in alphabetical order
- [x] Added Heroku Link(updated link)

Please Update the readme on route information if you have ideas on improvement or suggestions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,14 +133,102 @@ axios.request(options).then(function (response) {
...
+Pass in your desired `username`, `password`, and optional `role` as `user` or `admin` in a json format.
+
+ex.
+
+{
+  "username": "testUserName",
+  "password": "testPassword",
+}
+
+Optional adding "role": "user"
```
Comment (MuckT) [inline README.md]: No role for admin.

**Review by MuckT — CHANGES_REQUESTED** [review]

**Review by vbchomp — COMMENTED** [review]
Can you change the heroku deployment url to this - https://follow-the-nft.herokuapp.com/   ? We changed the production name to follow-the-nft and dev to follow-the-nft-dev.  
Other than that it looks good. I like the route listing at the bottom.

File: README.md (hunk 1 above)
Comment (fadnesscharlie) [reply README.md]: What do you mean?

Comment (fadnesscharlie) [reply README.md]: Oh you mean, add the role admin cause its not there

Comment (fadnesscharlie) [reply README.md]: - [x] added "role": "admin" example

File: README.md [hunk 2]
```diff
@@ -129,14 +133,105 @@ axios.request(options).then(function (response) {
...
+Pass in your desired `username`, `password`, and optional `role` as `user` or `admin` in a json format.
+
+ex.
+
+{
+  "username": "testUserName",
+  "password": "testPassword",
+}
+
+Optional adding:
```
Comment (MuckT) [inline README.md]: There isn't an admin role currently.

**Review by vbchomp — DISMISSED** [review]
Looks good for merge.

**Review by MuckT — DISMISSED** [review]

**Review by MuckT — APPROVED** [review]

**Review by vbchomp — APPROVED** [review]
looks good to merge

**Review by jeremywbrazell — APPROVED** [review]

**Pull request merged**